api-docs/rst/dev-guide/release-notes/as_20151204.rst

```diff
@@ -0,0 +1,22 @@
+API v1.1 release, December 08, 2015
+-----------------------------------
+
+What's new
+~~~~~~~~~~
+* Added a new **convergence** endpoint. You can submit a **POST** request to this endpoint to trigger convergence. Convergence implies that Autoscale tries to continuously converge to the desired state of the scaling group, instead of manipulating servers only once.
```

Comment:
@manishtomar 

Hi Manish,

I went ahead and incorporated all your feedback and committed all the changes to github. You should see them as part of the existing pull request. Please review the changes at your convenience. Once the content is merged, it will kick off a new content deployment.

Thanks, Constanze